PyU4V/migration.py

```diff
@@ -56,9 +59,12 @@ def get_array_migration_capabilities(self):
         :return: array_capabilities dict
         """
         array_capabilities = {}
-        target_uri = ("/{}/migration/capabilities/symmetrix".format(self.U4V_VERSION))
-        capabilities = self.common.get_request(target_uri, 'migration capabilities')
-        symm_list = capabilities.get('storageArrayCapability', []) if capabilities else []
+        target_uri = ("/{}/migration/capabilities/symmetrix".format(
+            self.U4V_VERSION))
+        capabilities = self.common.get_request(
+            target_uri, 'migration capabilities')
+        symm_list = capabilities.get('storageArrayCapability', []) \
+            if capabilities else []
```

Comment:
For extending commands across more than one line please use () instead of \. Example:

```
symm_list = (capabilities.get('storageArrayCapability', []) 
            if capabilities else [])
```

Reply:
Thank you for your feedback, I updated the code to avoid backslashes for line continuation. Adding parenthesis doesn't work with the "with foo as bar" syntax, but there's precedent for how to handle it elsewhere in the same file which I followed.

Reply:
Thanks Jason, we are in the process of reviewing our standards for the whole PyU4V library so we will have a good set of guidelines in the near future and consistency throughout.